ballerina-tests/tests/interceptors_test_common.bal

```diff
@@ -0,0 +1,176 @@
+import ballerina/http;
```

Comment:
Missing license header

Reply:
Will add that

Reply:
Fixed in [`c101943`](https://github.com/ballerina-platform/module-ballerina-http/pull/713/commits/c1019434762e8544d2380b9e543ceb9b74121a23)